VERSION_NUMBER

```diff
@@ -1 +1 @@
-1.8.0
+1.8.1
```

Comment:
Not sure whether it is needed since 1.8.1 is a patch.

Reply:
it is needed